### ISSUE TITLE
error code 0xc8a0f843

### ISSUE THREAD

**Issue by uzzyytgamer** [yellows8/ctr-httpwn#63]
hi every time i use ctr httpwn i always get the error code : 
ctr httpwn 0xc8a0f843 error 
'' reboot your system and try again. haxx setup failed : 0xc8a0f843. '' 

is there anyone who could help mewith this problem i have been trying for the last week to fix this problem and have looked on the internet but cant find the fix to it.

thanks

**Comment by RocketRobz** [comment]
If you have Luma CFW, is New 3DS CPU off in Luma settings?

**Comment by uzzyytgamer** [comment]
yes i have switched new3ds cpu off i have autoboot sysnand on , use sysnand firm if booting with R set as on, and show NAND or user string in system settings to on in the settings everything else is unchecked

**Comment by yellows8** [comment]
It's just BOSS, ignore it unless you really need that.

**Comment by uzzyytgamer** [comment]
i need to use ctr httpwn to go online in pokemon sun and moon but i cant is there any way i can fix it

**Comment by yellows8** [comment]
Ignore that error.

**Comment by uzzyytgamer** [comment]
ok thanks but how can i get on to my pokemon sun and moon online

**Comment by yellows8** [comment]
Old3DS/New3DS?

**Comment by uzzyytgamer** [comment]
new 3ds with luma v7

**Comment by yellows8** [comment]
"i cant" ?

**Comment by uzzyytgamer** [comment]
can i not go online without doing system update which will make me loose the luma custom firmware

**Comment by RocketRobz** [comment]
System Update does not make you lose Luma. Unless you want NTR's streaming feature, you can update.
